auth-web/src/components/auth/staff/SetupAccountForm.vue

```diff
@@ -0,0 +1,223 @@
+<template>
+  <div>
+    <div>
+      <v-form ref="directorSearchForm">
+        <v-alert type="error" v-show="errorMessage">{{errorMessage}}</v-alert>
+
+        <v-row>
```

Comment:
We'll need to reconsider this layout. We're not horizontally aligning these form fields